docs/tutorials/openfoam/spack-openfoam.yaml

```diff
@@ -114,6 +114,11 @@ deployment_groups:
     source: modules/scripts/startup-script
     settings:
       runners:
+      - type: shell
+        destination: remove_lustre_client.sh
+        content: |
+          #!/bin/bash
+          rm /etc/yum.repos.d/lustre-client.repo
```

Comment:
line 130,131, why are these lines needed now?
``` 
          mkdir -p /apps/openfoam
          chmod a+rwx /apps/openfoam
```

Reply:
please make sure you test it.

Reply:
Removed and ran the test.